addon/templates/components/frost-list-item-selection.hbs

```diff
@@ -1,7 +1,14 @@
-{{! Template for the frost-list-item-selection component }}
-
-{{frost-checkbox
-  checked=isSelected
-  hook=(concat hookPrefix '-checkbox')
-  size=size
-}}
+{{#if singleSelection}}
+  {{frost-radio-button 
+    checked=isSelected
+    hook=(concat hookPrefix '-radio-button')
+    size=size
+    value=(concat index '')
```

Comment:
Why do we need to concat an empty string?

Reply:
`frost-radio-button` requires a string for its value, but I figured it makes more sense to keep `index` as a number inside `frost-list-item-selection` in case we ever want to take further action based on index.

Reply:
Can you add a comment to the template explaining this?

Reply:
Please change the comment content to be

> frost-radio-button requires a string for "value", so we concat ''.  Approached this way so that "index" could remain a number as well in case we ever want to take further action based on its value.